Utilities/DataFlow/CMakeLists.txt

```diff
@@ -94,5 +97,8 @@ O2_GENERATE_TESTS(
 
 O2_GENERATE_MAN(NAME TimeframeReaderDevice)
 O2_GENERATE_MAN(NAME TimeframeWriterDevice)
+O2_GENERATE_MAN(NAME SubframeBuilderDevice)
 
 target_compile_options(test_TimeframeParser PUBLIC -O0 -g)
+target_compile_options(test_PayloadMerger01 PUBLIC -O0 -g)
+target_compile_options(test_SubframeUtils01 PUBLIC -O0 -g)
```

Comment:
I surrender to the crowd and will remove them... Democracy simply does not work... ;-)